### PR TITLE
Switch images over to HTTPS

### PR DESCRIPTION
**Pull request by gklopper**

### CODE REVIEW

File: common/conf/env/PROD.properties [hunk 1]
```diff
@@ -7,9 +7,9 @@ tag_indexes.bucket=aws-frontend-pressed
 # things like profile pages still work.
 assets.path=https://assets.guim.co.uk/
 
-images.path=//i.guim.co.uk
-static.path=http://static.guim.co.uk
-static.securePath=https://static-secure.guim.co.uk
+images.path=https//i.guim.co.uk
+static.path=https://static.guim.co.uk
+static.securePath=https://static.guim.co.uk
 staticSport.path=http://sport.guim.co.uk
```
Comment (gklopper) [inline common/conf/env/PROD.properties]: I will do sport too, the url is not yet HTTPS capable, but will be soon.

**Comment by OliverJAsh** [comment]
Nice :+1:

File: common/conf/env/PROD.properties [hunk 2]
```diff
@@ -7,9 +7,9 @@ tag_indexes.bucket=aws-frontend-pressed
 # things like profile pages still work.
 assets.path=https://assets.guim.co.uk/
 
-images.path=//i.guim.co.uk
-static.path=http://static.guim.co.uk
-static.securePath=https://static-secure.guim.co.uk
+images.path=https://i.guim.co.uk
+static.path=https://static.guim.co.uk
+static.securePath=https://static.guim.co.uk
 staticSport.path=http://sport.guim.co.uk
```
Comment (gklopper) [inline common/conf/env/PROD.properties]: Sport endpoint is not yet HTTPS capable - will be soon

**Comment by gklopper** [comment]
I will ship this tomorrow morning, its the kind of thing I like to see bed in for the day.

**Comment by sndrs** [comment]
:+1: wicked

File: common/conf/env/PROD.properties [hunk 3]
```diff
@@ -7,10 +7,10 @@ tag_indexes.bucket=aws-frontend-pressed
...
 assets.path=https://assets.guim.co.uk/
 
-images.path=//i.guim.co.uk
-static.path=http://static.guim.co.uk
-static.securePath=https://static-secure.guim.co.uk
-staticSport.path=http://sport.guim.co.uk
+images.path=https://i.guim.co.uk
+static.path=https://static.guim.co.uk
+static.securePath=https://static.guim.co.uk
+staticSport.path=https://sport.guim.co.uk
```
Comment (gklopper) [inline common/conf/env/PROD.properties]: HTTPS now works for sport

https://sport.guim.co.uk/football/crests/120/9.png

**Pull request merged**